app/editor/src/components/wysiwyg/CustomToolbar.tsx

```diff
@@ -0,0 +1,57 @@
+import { FaCode, FaExpandAlt, FaPaintBrush, FaRemoveFormat } from 'react-icons/fa';
```

Comment:
Moved a copy of the WYSIWYG into components folder.  Need to extract the old one at some point.